artifactory/services/movecopy.go

```diff
@@ -50,7 +50,10 @@ func (mc *MoveCopyService) MoveCopyServiceMoveFilesWrapper(moveSpecs ...MoveCopy
 	moveReaders := []*ReaderSpecTuple{}
 	defer func() {
 		for _, readerSpec := range moveReaders {
-			readerSpec.Reader.Close()
+			e := readerSpec.Reader.Close()
+			if err == nil && e != nil {
```

Comment:
Can be simplified to - 
```
if err == nil {
```

Reply:
its a code inside a deferred loop, so no checking e would override e